float64[16] F_T_EE
--------------------------


